src/main/kotlin/org/kamiblue/client/gui/clickgui/KamiClickGui.kt

```diff
@@ -89,4 +82,23 @@ object KamiClickGui : AbstractKamiGui<ModuleSettingWindow, AbstractModule>() {
             }
         }
     }
+
+    fun reorderModules() {
+        val allButtons = ModuleManager.modules
+            .groupBy { it.category.displayName }
+            .mapValues { (_, modules) -> modules.map { ModuleButton(it) } }
+
+        moduleWindows.forEach { window ->
+            window.children.clear()
+            allButtons[window.originalName]?.let { window.children.addAll(it.customSort()) }
+        }
+    }
+
+    private fun List<ModuleButton>.customSort(): List<ModuleButton> {
+        return when (ClickGUI.sortBy.value) {
+            ClickGUI.SortByOptions.CUSTOM -> this.sortedBy { -it.module.priorityForGui.value }
+            ClickGUI.SortByOptions.FREQUENCY -> this.sortedBy { -it.module.clicks.value }
```

Comment:
Use `.sortedByDecending` if you want it in reversed ordering

Reply:
Thanks